dotCMS/src/main/java/com/dotcms/system/event/local/type/publish/PublishEvent.java

```diff
@@ -0,0 +1,43 @@
+package com.dotcms.system.event.local.type.publish;
+
+import com.dotcms.publisher.business.PublishQueueElement;
+
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Object used to represent an event to be triggered among a push publishing process, ie: assets
+ * added to queue, push publishing starts, push publishing ends, etc
+ *
+ * @author nollymar
+ */
+public class PublishEvent {
+
+    private String name = null;
+    private List<PublishQueueElement> publishQueueElements = null;
+    private Date date;
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public List<PublishQueueElement> getPublishQueueElements() {
+        return publishQueueElements;
+    }
+
+    public void setPublishQueueElements(List<PublishQueueElement> publishQueueElements) {
+        this.publishQueueElements = publishQueueElements;
+    }
+
+    public Date getDate() {
+        return date;
+    }
+
+    public void setDate(Date date) {
+        this.date = date;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Store a copy of "date". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2384)